setup.py

```diff
@@ -1,8 +1,23 @@
 from setuptools import setup
 import os
 
+
 ignore_dir = ['.git']
 
+"""
+dirty fix for linchpin auto complete.
+Adds following expression to /etc/profile and sources it
+eval "$(_LINCHPIN_COMPLETE=source linchpin)"
+need to think of a better way to do the same.
+"""
+exp = "eval \"$(_LINCHPIN_COMPLETE=source linchpin)\""
+if exp not in open("/etc/profile", "r").read():
```

Comment:
This will not work in a virtualenv. I'd suggest not having this in the setup.py, but rather create a script that can be executed to generate the code needed. Also, system packages could handle this task.